deploy/local/live-csc/.env

```diff
@@ -48,7 +48,7 @@ TS_STANDARDSCRIPTS=../../../../ts_standardscripts/scripts
 TS_EXTERNALSCRIPTS=../../../../ts_externalscripts/scripts
 
 # lsstts/develop-env version
-LSSTTS_DEV_VERSION=c0017.000
+LSSTTS_DEV_VERSION=c0018.000
```

Comment:
Why didn't you use the `dev-cycle` variable in this repository?

Reply:
This is related to subsequent changes on this feature. At the beginning we started using the `LSSTTS_DEV_VERSION`, after this was done, our partners from the observatory also started using this feature with `cycle` environment variable, only for use in production deployment. As this change was added, I preferred to add the `dev-cycle` environment variable to be used by the `docker-compose.yml` of the specific repos (not the integration tools). This was a previous discussion and our partners of the telescope would prefer to have similar names, so **yes** we should use the `dev-cycle` name instead of the `LSSTTS_DEV_VERSION` name, the thing is that this will obligate us to **change every Dockerfile used for development of each repo that uses the image** (i.e. producer, commander, simulator)! Maybe we could note this down and add it on a future PR on each repo, what do you think?

Reply:
Ok, I understood. I agree with you.